### DISCUSSION TITLE
icub console backup needs?

### DISCUSSION THREAD

**Discussion by pfdominey** [robotology/community#531] [Q&A] [answered]
Greetings,
We are about to send iCub01Lyon to IIT for equipment and maintenance.  (He is happy - has a sore right arm!)  We will send the console computer with it.  I have started to backup the computer.  It is linux 3.16.04 debian 4.8.4-1 2016.  I was to backup /usr and /home. Is this sufficient?  
Probably there is Wysiwyd code from inserm on a github.  Do you know if this is still available?  
Thanks
cheers
Peter

**Comment by pattacini** [comment, marked as answer]
Hi @pfdominey 

The `WYSIWYD` repo is hosted at [`robotology-legacy/wysiwyd`](https://github.com/robotology-legacy/wysiwyd).

Probably, the console will be regenerated software-wise with a fresh installation of the latest OS and iCub distro; therefore, the backup of `/usr` won't be necessary, but it won't certainly harm.

The backup of `/home` is a good thing to do as it contains your local YARP settings (apps, contexts...).
Don't think there's more than this.

cc @mbrunettini
